src/java/picard/sam/SortSam.java

```diff
@@ -51,16 +51,24 @@
 )
 public class SortSam extends CommandLineProgram {
     static final String USAGE_SUMMARY = "Sorts a SAM or BAM file.  ";
-    static final String USAGE_DETAILS = "This tool sorts the input SAM or BAM file by coordinate, queryname  or some other property " +
-            "of the SAMRecord. Input and output formats (SAM or BAM) are determined by the file extension." +
-            "<br />" +
+    static final String USAGE_DETAILS = "This tool sorts the input SAM or BAM file by coordinate, queryname (QNAME), or some other property " +
+            "of the SAM record. The SortOrder of a SAM/BAM file is found in the SAM file header tag (@HD) in the field labeled (SO).  " +
```

Comment:
"SAM file header tag (@HD) in the field labeled (SO)" -> "SAM file header line starting with @HD in the field labeled SO"